backend/fms_core/filters.py

```diff
@@ -75,6 +76,16 @@ def is_pooled_filter(self, queryset, name, values):
         bool_value = (values == 'true')
         return queryset.filter(is_pooled=bool_value)
 
+    def metadata_filter(self, queryset, name, values):
+        condition = Q()
+        # Metadata is of form: 'name1__value1, name2__value2, name2__Value3'
+        for metadatum in values.split(','):
+            if metadatum:
+                name_value = metadatum.split('__')
```

Comment:
There may be more than an instance of "__" since the value part is not controlled. We should add a validation to the metadata name attribute in order to prevent "__" to appear in the name. That way you can split on the first occurrence of "__" in the querystring value (not on all or you'll get an array greater than 2 possibly).